common/config/rush/command-line.json

```diff
@@ -16,6 +16,7 @@
       "name": "prettier",
       "commandKind": "global",
       "summary": "Used by the pre-commit Git hook. This command invokes Prettier to reformat staged changes.",
+      "safeForSimultaneousRushProcesses": true,

```

Comment:
Not sure this is safe

Reply:
![CleanShot 2023-07-10 at 08 06 00](https://github.com/priceline/design-system/assets/3260642/5b1844c6-d73c-4991-b834-499a5ddf7a1c)

I got it from whatever's current in the Rush docs

Reply:
👍